ci/jenkinsfiles/release.groovy

```diff
@@ -0,0 +1,324 @@
+                  -------------------------------------------------
+                """
+                if (env.DRY_RUN_RELEASE == 'false') {
+                  pipelineLib.uploadPackage("${VERSION}", 'connect-prod', "${CONNECT_PROD_URL}")
+                }
+              }
+            }
+          }
+          post {
+            always {
+              archiveArtifacts (
+                allowEmptyArchive: true,
+                artifacts: 'nuxeo-coldstorage-package/target/nuxeo-coldstorage-package-*.zip'
+              )
+            }
+          }
+        }
+        stage('Git Push') {
+          steps {
+            container('maven') {
+              echo """
+                --------------------------
+                Git push ${TAG}
+                --------------------------
+              """
+              script {
+                if (env.DRY_RUN_RELEASE == 'false') {
+                  pipelineLib.gitPush("${TAG}")
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    stage('Release and Bump reference branch') {
+      when {
+        allOf {
+          not {
+            environment name: 'DRY_RUN', value: 'true'
+          }
+        }
+      }
+      steps {
+        container('maven') {
+          script {
+            //cleanup files updated by other stages
+            sh 'git reset --hard'
+            // increment minor version
+            String nextVersion =
+              sh(returnStdout: true, script: "perl -pe 's/\\b(\\d+)(?=\\D*\$)/\$1+1/e' <<< ${CURRENT_VERSION}").trim()
+            echo """
+              -----------------------------------------------
+              Update ${REFERENCE_BRANCH} version from ${CURRENT_VERSION} to ${nextVersion}
+              -----------------------------------------------
+            """
+            pipelineLib.gitCheckout("${REFERENCE_BRANCH}")
+            pipelineLib.updateVersion("${nextVersion}")
+            pipelineLib.gitCommit("Release ${VERSION}, update ${CURRENT_VERSION} to ${nextVersion}", '-a')
+            if (env.DRY_RUN_RELEASE == 'false') {
+              pipelineLib.gitPush("${REFERENCE_BRANCH}")
+            }
+          }
+        }
+      }
+    }
+  }
+  post {
+    always {
+      script {
+        currentBuild.description = "Release ${VERSION} from build ${params.BUILD_VERSION}"
+      }
+    }
+    success {
+      script {
+        // update Slack Channel
+        String message = "Successfully released ${VERSION} from build ${params.BUILD_VERSION}: ${BUILD_URL} :tada:"
+        pipelineLib.setSlackBuildStatus("${SLACK_CHANNEL}", "${message}", 'good')
+      }
+    }
+    unsuccessful {
+      script {
+        // update Slack Channel
+        String message = "Failed to release ${VERSION} from build ${params.BUILD_VERSION}: ${BUILD_URL}"
+        pipelineLib.setSlackBuildStatus("${SLACK_CHANNEL}", "${message}", 'danger')
+      }
+    }
+  }
+}
```

Comment:
Perhaps, I am missing something. But why those three files are part of the cleanup/format commit.

Reply:
Hello
will be removed ( and create another PR  for the release)